src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/SlimefunProfiler.java

```diff
@@ -217,6 +221,11 @@ private void finishReport() {
 
         totalElapsedTime = timings.values().stream().mapToLong(Long::longValue).sum();
 
+        // We log how much ms has been ticked and how many ticks have passed.
```

Comment:
```suggestion
        // We log how many ms have been ticked and how many ticks have passed.
```

Reply:
Not even sure this would be the best way to express it but eh, I'm not the English one 😛

Reply:
Improved it further

Reply:
'Have been passed' means something different :smile: I think 'have passed' is correct but I don't wanna annoy you too much :smile: